src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        if (Storage.fruits.containsKey(transaction.getFruit())
+                && transaction.getOperation() == FruitTransaction.Operation.RETURN) {
+            int newValue = Storage.fruits.get(transaction.getFruit())
+                    + transaction.getQuantity();
+            Storage.fruits.put(transaction.getFruit(), newValue);

```

Comment:
Direct access to the storage map is used instead of encapsulating the access in methods. This can lead to issues if the storage implementation changes and does not use a map.

Reply:
ok, done